corpus/pages/views.py

```diff
@@ -25,3 +26,15 @@ def about_us(request):
             "socieites": societies,
         },
     )
+
+
+def compsoc(request):
+    sigs = SIG.objects.all()
+
+    return render(
+        request,
+        "pages/sig.html",
+        {
+            "sigs": sigs,
+        },
+    )
```

Comment:
Why is this called `compsoc`. Pass the name of the SIG in the URL and query based on that.

Reply:
done